OrchardCore.Commerce.Tests/AmountTests.cs

```diff
@@ -1,11 +1,9 @@
 using System;
+using Money;
+using Money.Abstractions;
 using Newtonsoft.Json;
-using OrchardCore.Commerce.Abstractions;
-using OrchardCore.Commerce.Money;
 using Xunit;
 
-using static OrchardCore.Commerce.Money.Currency;
```

Comment:
Please keep the `using static`.

Reply:
`OrchardCore.Commerce.Money.Currency` no longer exists, compile error occurs if I keep this in.

You mentioned earlier to move the data type to it's own assembly with no reference to OrchardCore, I did that, therefore this is no longer needed.  If I misunderstood, can you please clarify.

Reply:
Keep the using static, but with the correct new namespace. This is about keeping the currency names short.

Reply:
Sorry, misunderstood.  Fixed.